hr_holidays_leave_repeated/views/hr_leave.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="hr_leave_repeated_view_form" model="ir.ui.view">
+        <field name="model">hr.leave</field>
+        <field name="inherit_id" ref="hr_holidays.hr_leave_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='holiday_status_id']" position="after">
+                <div class="col-4 mr-0">
```

Comment:
```suggestion
                <div class="col-5 mr-0">
```

This fixes the look in case the form is opened from the calendar, by clicking on Edit button.